test/e2e/e2e_duplicatepods_test.go

```diff
@@ -138,6 +139,10 @@ func TestRemoveDuplicates(t *testing.T) {
 			if err != nil || len(evictionPolicyGroupVersion) == 0 {
 				t.Fatalf("Error creating eviction policy group %v", err)
 			}
+
+			eventBroadcaster, eventRecorder := utils.GetRecorderAndBroadcaster(ctx, clientSet)
```

Comment:
Given the e2e test does not check the events recorded during testing the `eventRecorder` can be replaced by [NewFakeRecorder](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/tools/events/fake.go#L41) which does not need any `eventBroadcaster.Shutdown()`.

Reply:
Benefit of the fake recorder is no go routine inside `Eventf`.